### ISSUE TITLE
Style toggle

### ISSUE THREAD

**Issue by lwrubel** [sul-dlss/hungry-hungry-hippo#334]
The toggle used in Authors tab (and eventually elsewhere) has button-like edges but per the designs it should be more rounded. If you have further feedback, @astridu, please add here. 

Existing toggle in #303: 
![Screenshot 2024-12-11 at 5 04 39 PM](https://github.com/user-attachments/assets/be410b57-c263-4832-a097-80efa36a7ba2)

Toggle in designs:
![Screenshot 2024-12-11 at 5 05 18 PM](https://github.com/user-attachments/assets/36cc6e9f-3c61-4899-aad0-29075628df31)

**Comment by lwrubel** [comment]
Noting that there are toggles in the wild that look like the "radio button group" style currently implemented. From Github:

![Screenshot 2024-12-11 at 5 08 46 PM](https://github.com/user-attachments/assets/8dac544e-bcdd-4de3-b87f-894e2e6aa6e5)

**Comment by astridu** [comment]
So sorry. You are right, the more square one is out in production, but DLUX has not decided on the style of the toggle. I will get back to you by end of week w/our decision. Thank you for your patience!

**Comment by lwrubel** [comment]
The square one is just from a PR--are there toggles like this in production? No rush on a style decision for H3.

**Comment by astridu** [comment]
Yes, there is a square one in EarthWorks. It's used to show an expanded vs. compact view of the search results. 
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/8d6abc73-6228-4883-97cb-cd0d845c634e" />

**Comment by astridu** [comment]
@lwrubel I spoke w/DLUX, and we decided on the following for use of radio button vs. square toggle vs. pill toggle (the more rounded one in my mockups):

- Radio buttons: select one from a set of options.

- Toggle: turn features on or off, changes presentation.

- - Rounded corner square for presentation changes

- - Pill shape when set in a form


So for the ones that appear in H3, since they are set in a form, they will be the pill-shape. Let me know if you have more questions.

**Comment by amyehodge** [comment]
@astridu @lwrubel Is there something that needs to be done for this ticket or can we close this? Thanks.

**Comment by lwrubel** [comment]
The toggles are still currently rounded corners rather than pills, so to stay in alignment with the design, I think they need to be styled as Astrid describes.

**Comment by justinlittman** [comment]
@astridu Can you provide a specification or example for the rounding?

**Comment by astridu** [comment]
It's a 15px radius for the rounded corner. Does that help?

**Comment by justinlittman** [comment]
Thanks -- I think so.
